gc/policy/policy.go

```diff
@@ -0,0 +1,217 @@
+package policy
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/metadata"
+	"github.com/containerd/containerd/plugin"
+)
+
+type config struct {
```

Comment:
I'd prefer if this was exported, will avoid people generating the `containerd.toml` to have to duplicate the struct in their code.